source/retryable-reads/tests/estimatedDocumentCount-4.9.yml

```diff
@@ -0,0 +1,64 @@
+runOn:
+    -
+        minServerVersion: "4.9.0"
```

Comment:
You can make this a one-liner as you did in the unified CRUD test:

```
- minServerVersion: "4.9.0"
```

Likewise for other instances where it spans multiple lines.

Reply:
Done!